userspace/falco/app/actions/configure_interesting_sets.cpp

```diff
@@ -73,6 +74,25 @@ static void select_event_set(falco::app::state& s, const libsinsp::events::set<p
 			+ ") syscalls in rules: " + concat_set_in_order(rules_names) + "\n");
 	}
 
+	/* Load PPM event codes needed by plugins with parsing capability */
+	libsinsp::events::set<ppm_event_code> plugin_ev_codes;
+	for (const auto &p : s.offline_inspector->get_plugin_manager()->plugins())
+	{
```

Comment:
shouldn't libs code be formatted in a different way now?

Reply:
What do you mean? This is Falco :laughing:

Reply:
-ENOBRAIN

Reply:
dead :rofl: